packages/core/src/interfaces/axis-scales.ts

```diff
@@ -124,6 +124,15 @@ export interface AxisOptions extends BasedAxisOptions {
 	 * option for percentage axis scale
 	 */
 	percentage?: boolean;
+	/**
+	 * should be set to `true` for the
+	 * left axis to be the primary axis
+	 */
+	main?: boolean;
+	/**
+	 * used to map data on the secondary axis
+	 */
+	correspondingDatasets?: Array<string>;
```

Comment:
Hi @metonym, we should probably find a way to have this only affect combo charts, since they mostly wouldn't apply to others

Reply:
https://github.com/carbon-design-system/carbon-charts/commit/8b492e0329829cc9619faea88605def7e463b933

- created a `ComboChartAxisOptions` interface
- updated `ComboChartOption.axes` to use `ComboChartAxisOptions`

Reply:
@theiliad, I don't get it. Why shouldn't I use these properties in `ScatterChartOptions` as described in my example in https://github.com/carbon-design-system/carbon-charts/issues/1413? For me, this issue doesn't seem to be fixes as I was trying to use them in a `ScatterChartOptions`.